ravenframework/Steps/IOStep.py

```diff
@@ -111,6 +111,8 @@ def _localInitializeStep(self,inDictionary):
       elif isinstance(inDictionary['Input'][i], (Models.ROM, Models.ExternalModel)):
         # ... file
         if isinstance(outputs[i],Files.File):
+          if 'Pyomo' == outputs[i].getType():
+            self.actionType.append('MODEL-PYOMO')
```

Comment:
I would to change the ```if 'Pyomo' == outputs[i].getType()''' to ```if 'PYOMO' == outputs[i].getType().upper()'''

Reply:
Changed to suggested syntax.